util/Migrator/DbScripts/2025-12-02_AddCipherArchives.sql

```diff
@@ -0,0 +1,781 @@
+-- Add new JSON column for Archives (similar to Favorites/Folders pattern)
+IF NOT EXISTS (
+    SELECT 1
+    FROM sys.columns
+    WHERE object_id = OBJECT_ID(N'[dbo].[Cipher]')
+    AND name = 'Archives'
+)
+BEGIN
+    ALTER TABLE [dbo].[Cipher]
+    ADD [Archives] NVARCHAR(MAX) NULL;
+END;
+GO
+
+-- Update CipherDetails function to use JSON column approach
+IF OBJECT_ID('[dbo].[CipherDetails]') IS NOT NULL
+BEGIN
+    DROP FUNCTION [dbo].[CipherDetails];
+END
+GO
+
+CREATE FUNCTION [dbo].[CipherDetails](@UserId UNIQUEIDENTIFIER)
```

Comment:
We can just use a CREATE OR ALTER FUNCTION here instead of first dropping the function and then creating it